docs/README.md

```diff
@@ -10,6 +10,7 @@ Follow the links below to their respective API usage examples:
 - [Clips](clips_docs.md)
 - [Entitlement Grants](entitlement_grants_docs.md)
 - [Games](games_docs.md)
+- [Logging](logging_docs.md)
```

Comment:
Given that we're moving the logging docs to this same file, this line can be dropped.